### DISCUSSION TITLE
Vehicle fire inside tunnel - Numerical instability error

### DISCUSSION THREAD

**Discussion by jkjena1955** [firemodels/fds#13695] [Q&A]
Hi All,

I am simulating vehicle fire inside tunnel. I have taken fuel as N-Heptane (engine oil equivalent), PVC for plastic material and Urethane as seat material. I have used complex combustion model in FDS input for all the 3 fuel materials. My objective is to find the peak HRR, Peak Temperature and the time taken to reach both.  

Since, i have workstation for simulation that has 2 sockets, 44 cores and 88 logical cores. Thus, limiting the tunnel length to 20 m for simulating two truck (9.8 m length) placed in tandem. The fire gets ignited from the engine oil (N-Heptane) through a bunch of ignitor particles at 2000 deg C. The trucks front dash boards and inner liner are modelled as PVC. The seats are in Urethane. Trucks are loaded with PVC plastic slabs in arrangement . The simulation is running for 10 to 30 sec. Then it stops with numerical instability error.

I am sharing the FDS input file and screenshot of the error.

Please suggest ways to get rid of the numerical instability error in run time.

[t6t6.zip](https://github.com/user-attachments/files/17634160/t6t6.zip)

Best Regards,
Jena

**Comment by rmcdermo** [comment]
Using high temperature ignitor particles can be problematic.   I would opt for a different ignition method.  Also, update to the latest code... lots of bug fixes between 6.7 and now.

**Comment by jkjena1955** [comment]
Hi Randy,

Thanks for the response. Can you please advise on the different ignition method that i can try. Secondly, I tried to run the code in latest FDS version. The ignition does not happen with multiple fuels with complex combustion. Can you please look into the code and tell the syntax change required to make it run in the latest version.

Thanks,
Jena

**Comment by rmcdermo** [comment]
Ignition is the wild west.  You can try INIT with HRRPUV, for example, and apply a RAMP_Q to the HRRPUV.

Sorry I don't have time right now to debug the inputs for you.  My advice is to start very simple and work your way up.  I don't like that you are not using default SPEC.  Ignition and other phenomena you are trying to model depend on good thermophysical properties, which you are not specifying.  With latest code, you can use multiple reactions with simple chemistry.  Start by getting your heptane to ignite.

**Reply by jkjena1955** [reply]
Since, I am using PVC that contains Chlorine, simple chemistry with predefined species does not work. Simple chemistry and complex chemistry does not go together in FDS as it impacts the background air. Can you please suggest a material that contains (C,H,O,N) only that is equivalent to PVC?

**Comment by mcgratta** [comment]
I am running your case, but you middle mesh is very large and it will take some time to reach the point of instability. I would advise that your create a more simple case that does not use such large meshes. I generally run dozens of simulations while deciding what final parameters to use.

**Reply by jkjena1955** [reply]
Thanks Kevin for puting your effort to run the simulation! 10 cm mesh is taken for the burning zone. Are you suggesting that i should go for 5 cm mesh size? I afraid, it won't run in my system. 

Please let me know your suggestion post run.

Regards,
Jena

**Reply by mcgratta** [reply]
No, I mean the opposite. If you want to use 10 cm cells, I suggest you break up the 2 million cell mesh so that all your meshes have a similar number of cells. The speed of the calculation is dependent on the largest mesh.

**Reply by jkjena1955** [reply]
I am using 10 cm for the burning volume and 20 cm for the peripherals. Now i will give a run with similar cell numbers in the 3 meshes.

Did you observe the numerical instability in the run? What are you suggesting about the possible reason?

**Comment by mcgratta** [comment]
I started the case with the latest source code. It is still running at 140 s.  Try running the case with the [latest nightly build](https://github.com/firemodels/test_bundles/releases/tag/FDS_TEST)

**Reply by jkjena1955** [reply]
What do you mean by nightly build?

**Reply by drjfloyd** [reply]
https://github.com/firemodels/test_bundles

**Reply by jkjena1955** [reply]
thanks

**Comment by mcgratta** [comment]
This case is still running at 290 s. I am going to stop it. Try the latest version of FDS and report back if the job runs.

**Reply by jkjena1955** [reply]
thanks..let me try it
